app/components/listing/Notes.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import ReactMarkdown from 'react-markdown';
 
 export default function Notes(props) {
-  return (
+  return props.notes && (
```

Comment:
It looks like I made a conflicting change in https://github.com/ShelterTechSF/askdarcel-web/pull/795, where I hoisted the guards to the calling component. I'm guessing this will require updating the tests you added here.